meinberlin/react/livequestions/StatisticsBox.jsx

```diff
@@ -42,83 +42,79 @@ export default class StatisticsBox extends React.Component {
 
     return (
-      <div className="module-content--light">
-        <div className="container">
-          <div className="offset-lg-2 col-lg-8">
-            {this.props.categories.length > 0 &&
-              <div>
-                <h3>{categoriesAnsweredTag}</h3>
-                <div className="list-item">
-                  {this.props.categories.map((category, index) => {
-                    const countPerCategory = this.countCategory(category)
-                    const style = { width: countPerCategory + '%' }
-                    return (
-                      <div key={index} className="u-spacer-bottom-half">
-                        <div className="progress">
-                          <div
-                            className="progress-bar" style={style} role="progressbar" aria-valuenow="25" aria-valuemin="0"
-                            aria-valuemax="100"
-                          />
-                          <span className="progress-bar__stats">&nbsp;{category}&nbsp;{countPerCategory}%</span>
-                        </div>
-                      </div>
-                    )
-                  })}
-                </div>
-              </div>}
-            <h3>{questionAnsweredTag}</h3>
-            {this.props.isModerator
-              ? (
-                <div className="list-group">
-                  {this.state.answeredQuestions.map((question, index) => {
-                    return (
-                      <QuestionModerator
-                        updateQuestion={this.updateQuestion.bind(this)}
-                        displayIsOnShortlist={false}
-                        displayIsLive={false}
-                        displayIsHidden={false}
-                        displayIsAnswered={question.is_answered}
-                        removeFromList={this.removeFromList.bind(this)}
-                        key={question.id}
-                        id={question.id}
-                        is_answered={question.is_answered}
-                        is_on_shortlist={question.is_on_shortlist}
-                        is_live={question.is_live}
-                        is_hidden={question.is_hidden}
-                        category={question.category}
-                        likes={question.likes}
-                      >
-                        {question.text}
-                      </QuestionModerator>
-                    )
-                  })}
-                </div>
+      <div className="container">
+        {this.props.categories.length > 0 && (
+          <>
+            <h2>{categoriesAnsweredTag}</h2>
+            <div className="modul-card card">
+              <h3>{categoriesTag}</h3>
+              {this.props.categories.map((category, index) => {
+                const countPerCategory = this.countCategory(category)
+                const style = { width: countPerCategory + '%' }
+                return (
+                  <div className="live_question__progress" key={index}>
```

Comment:
I know it's been like that in the code before, but could you change it to use something other than `index`?

Reply:
Thanks, I've now added `category_dict` here, as the keys are the `category` `id`s: https://github.com/liqd/a4-meinberlin/pull/5750/files#diff-2d2b8a5601ea372ea7c39a1e981837692b43614c93d46ab5c0a13f39c8196dd3R54